### ISSUE TITLE
Menu, MenuItem, MenuDivider

### ISSUE THREAD

**Issue by flake** [poetic/react-material-ui#2]
Please add new MenuItem from lib/MenuItem

because some issues from npm show that new menuitem should be used from /material-ui/lib/menus/menu-item [#1346](https://github.com/callemall/material-ui/issues/1346)

**Comment by flake** [comment]
@poetic-marketing How do I get the MenuItem???

**Comment by codeNgamer** [comment]
@flake the menuItem should be accessible using:

``` javascript
let MenuItem =  mui.MenuItem
//alternatively using destructuring
const {MenuItem} =  mui
```

**Comment by codeNgamer** [comment]
@flake can you confirm everything works for you now so I can close this issue. Thanks

**Comment by flake** [comment]
@codeNgamer NO, i dropped your package. MenuItem has different export from node package which is mention in previous post  /material-ui/lib/menus/menu-item

**Comment by codeNgamer** [comment]
@flake ahh i see what you mean, sorry for the confusion. I'll update, test and publish shortly. Again, sorry about that

**Comment by james-cordeiro** [comment]
is this updated now - I'm not seeing any progress into requiring the new MenuItem but I'm unsure how to do the pull request my myself.... Hoping to get this in ASAP if not appropriate hacky solution or otherwise. Thanks
